modules/custom/openy_upgrade_tool/openy_upgrade_tool.install

```diff
@@ -55,22 +90,30 @@ function openy_upgrade_tool_update_8004(&$sandbox) {
 
   $configFactory = \Drupal::configFactory();
   $entityTypeManager = \Drupal::entityTypeManager();
+  $upgrade_log_manager = \Drupal::service('openy_upgrade_log.manager');
   $logger_entities = $entityTypeManager
     ->getStorage('logger_entity')
     ->loadMultiple($ids);
```

Comment:
we can't log all entities. Let's have array_slice by 50 at most. Take care about memory, please, it is not infinite.

Reply:
@podarok please see code above, it already in chunks by 20 items:
```php
  $ids = \Drupal::entityQuery('logger_entity')
    ->condition('type', 'openy_config_upgrade_logs')
    ->condition('id', $sandbox['current'], '>')
    ->range(0, 20)
    ->sort('id')
    ->execute();
```